fastddsspy_participants/src/cpp/model/DataStreamer.cpp

```diff
@@ -54,6 +67,7 @@ bool DataStreamer::activate(
 void DataStreamer::deactivate()
 {
     std::unique_lock<std::shared_timed_mutex> _(mutex_);
+    activated_ = false;
```

Comment:
Should activated_all_ also be set to false?

Reply:
It is not needed, as `activated_all_` is set before used in `activate` and `activate_all`, so it wont be used before set